praktikumsplaner-backend/src/main/resources/db/migration/V8__Add_relation_from_Praktikumsstelle_to_Meldezeitrum.sql

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE AUSBILDUNGSPRAKTIKUMSSTELLE ADD COLUMN meldezeitraumID varchar(36) not null default 0;
```

Comment:
`0` do not work too: `ERROR: insert or update on table "ausbildungspraktikumsstelle" violates foreign key constraint "ausbildungspraktikumsstelle_meldezeitraumid_fkey"
  Detail: Key (meldezeitraumid)=(0) is not present in table "meldezeitraum".`